frappe/public/js/frappe/list/list_sidebar.js

```diff
@@ -49,12 +48,27 @@ frappe.views.ListSidebar = class ListSidebar {
 
 	setup_views() {
 		var show_list_link = false;
+		var cur_doc=this.doctype;
+		var check = false;
+		frappe.call({
+			method: "frappe.core.page.calendar.calendar.get_all_calendars",
+			type: "GET"
+		}).then(r => {
+			$.each(r["message"], function( index, value ) {
+				if(cur_doc == value){
+					check = true;
+					return false;
```

Comment:
i think this isn't needed, or can be something else

Reply:
yeah removed this check